connectors/github/README.md

```diff
@@ -1,4 +1,11 @@
-# GitHub Connector
+# GitHub Connectors
 
+## GitHub API
```

Comment:
nit: we can also explicitly say that this is an outbound connector, and webhook is the inbound one